### DISCUSSION TITLE
Configurable cache prefix

### DISCUSSION THREAD

**Discussion by lloricode** [opcodesio/log-viewer#371] [Ideas]
Hi, i was wondering if posible to add custom cache prefix, or anything that can be ignore on monitoring like laravel pulse, 

![Screenshot_20240512_004304_Brave](https://github.com/opcodesio/log-viewer/assets/8251344/dcf8b591-ad20-4b8b-b1c3-3be34849f232)

It almost a hundred cache event trigered

Hope some one give an idea, thank you

**Comment by lloricode** [comment]
I did see this, https://github.com/opcodesio/log-viewer/blob/main/src/Utils/GenerateCacheKey.php#L34

**Comment by arukompas** [comment]
Hey @lloricode 

In Pulse you can group similar cache keys, and all Log Viewer keys start with "lv:<version>:"

So try adding the grouping config:

https://laravel.com/docs/11.x/pulse#cache-interactions-recorder

**Reply by lloricode** [reply]
Yeah already did before read this, and it works like a charm 😁, thank you for the response btw

**Reply by lloricode** [reply]
Hi @arukompas also can you consider to dynamically set cache prefix? Incase some conflict

**Reply by arukompas** [reply]
Does sound like a valid request. I'll have look into it!

**Reply by arukompas** [reply]
hey @lloricode , the option is now available in Log Viewer **v3.10.0** :) 

https://log-viewer.opcodes.io/docs/3.x/configuration/result-caching#cache-prefix

**Comment by arukompas** [comment]
FYI, cache key prefix is now configurable since **v3.10.0**:

https://log-viewer.opcodes.io/docs/3.x/configuration/result-caching#cache-prefix

**Reply by lloricode** [reply]
Thank you
